.github/workflows/vale.yml

```diff
@@ -0,0 +1,36 @@
+name: Review Docs
+on:
+  pull_request:
+    branches: main
+
+jobs:
+  review-docs:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    outputs:
+      files: ${{ steps.get-modified-files.outputs.files }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Get modified files
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        id: get-modified-files
+        run: |
+          changed_files=$(gh pr diff --name-only ${{ github.event.pull_request.number }} | grep '\.md$' || true)
```

Comment:
@0xLucca, just reviewing this line, maybe worth adding support to `.pages` files to avoid typos there as well:
```bash
changed_files=$(gh pr diff --name-only ${{ github.event.pull_request.number }} | grep '\.md$|.pages$' || true)
```

WDYT?

Reply:
Yeah would also like it to catch capitalization of page titles if possible